src/Helpers/approval/approval-helper.js

```diff
@@ -0,0 +1,3 @@
+import { APPROVAL_API_BASE } from '../../Utilities/Constants';
+
+export const getApprovalWorkflow = () => fetch(`${APPROVAL_API_BASE}/workflows`).then(data => data.json());
```

Comment:
nitpick - plural here: getApprovalWorkflows?

Reply:
Not for this PR - but we should use the generated js client code when that is fixed.

Reply:
> Not for this PR - but we should use the generated js client code when that is fixed.

Yep i know but lately i am constantly switching between client/custom calls because of some missing attributes. I am now planing to do some small "audit" every couple of weeks and replace the custom calls in groups.